android/java/org/chromium/chrome/browser/crypto_wallet/fragments/onboarding/OnboardingCreatingWalletFragment.java

```diff
@@ -41,6 +50,16 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     @Override
     public void onResume() {
         super.onResume();
+
+        final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
```

Comment:
let's better use chromium's scheduler and thread pool https://source.chromium.org/chromium/chromium/src/+/main:base/android/java/src/org/chromium/base/task/PostTask.java;l=95?q=PostTask.java&ss=chromium%2Fchromium%2Fsrc.
An example of usage is here https://github.com/brave/brave-core/blob/master/android/java/org/chromium/chrome/browser/app/BraveActivity.java#L2173.
I know we have places where we still use system `Handler` and `Looper`. But they ideally would need to be refactored to use chromium's queue as we really mess it up with non controlled threads and schedulers. It would be a good move to start using upstream's functionality in new implementation.

Reply:
Studied better Chromium scheduler, and ended up using `USER_BLOCKING` trait as `UI_DEFAULT` was not fast enough to disable the delay.